### PR TITLE
Create Events Pages “hacktoberfest-2020/index-md”

### PR DESCRIPTION
**Pull request by isaac-pulumi**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/events/hacktoberfest-2020/index-md.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+This year, Pulumi is participating in Hacktoberfest with three areas where the community at large can collaborate to build new open source capabilities:
+
+1. **Native Pulumi Providers** - Want Pulumi to work better with your favorite cloud services? We'll show you how to get started building providers that let Pulumi users take full advantage of these services.
+2. **Operator and Automation API** - Learn how to use Pulumi's new Kubernetes Superpowers to build truly automated deployments for your workloads.
+3. **Reusable Infrastructure Libraries** - Create new abstractions and infrastructure building blocks using Pulumi.
+
+## How to get started
+
+1. **[Register](https://hacktoberfest.digitalocean.com)** on the Hacktoberfest site for access to get credit for your open source contributions.
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:39]: @leezen I'll update this link after we publish on Hacktoberfest - it's a chicken and egg problem (both pages need a link referring to each other).

File: content/events/hacktoberfest-2020/index-md.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  time: 9:00 AM - 11:00 AM
+  cost: $0
+  end_date: 2020-10-23
+  location: VIRTUAL
+  type:
+    - Hackathon
+  description: Join the Pulumi team and members of the community as we hack new
+    integrations, automation, and infrastructure libraries!
+type: events
+url_slug: workshop-amsterdam-eu-kubecon-2020
```
Comment (leezen) [inline content/events/hacktoberfest-2020/index-md.md:24]: Needs updating

File: content/events/hacktoberfest-2020/index-md.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+meta_desc: Join Pulumi at our Infrastructure As Code Workshop at KubeCon EU and
+  learn more about cloud programming, infrastructure as code, and many other
+  topics.
```
Comment (leezen) [inline content/events/hacktoberfest-2020/index-md.md:4]: Needs updating

File: content/events/hacktoberfest-2020/index-md.md (hunk 2 above)
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:24]:
```suggestion
url_slug: hacktoberfest-2020
```

File: content/events/hacktoberfest-2020/index-md.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+meta_desc: Join Pulumi at our Infrastructure As Code Workshop at KubeCon EU and
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:2]:
```suggestion
meta_desc: Join Pulumi for Hacktoberfest and help us build open source integrations, automation and libraries.
```

File: content/events/hacktoberfest-2020/index-md.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  learn more about cloud programming, infrastructure as code, and many other
+  topics.
+title: Hacktoberfest 2020
+subtitle: Help us build new open source capabilities for Pulumi
+unlisted: false
+redirect_to: https://hacktoberfest.digitalocean.com
+event:
+  eventbrite_url: " "
+  registration_url: https://hacktoberfest.digitalocean.com
+  calendly_url: " "
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:12]:
```suggestion
```

File: content/events/hacktoberfest-2020/index-md.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+---
+meta_desc: Join Pulumi at our Infrastructure As Code Workshop at KubeCon EU and
+  learn more about cloud programming, infrastructure as code, and many other
+  topics.
+title: Hacktoberfest 2020
+subtitle: Help us build new open source capabilities for Pulumi
+unlisted: false
+redirect_to: https://hacktoberfest.digitalocean.com
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:8]:
```suggestion
```

File: content/events/hacktoberfest-2020/index-md.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+event:
+  eventbrite_url: " "
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:10]:
```suggestion
```

File: content/events/hacktoberfest-2020/index-md.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+  registration_url: https://hacktoberfest.digitalocean.com
+  calendly_url: " "
+  eventbrite_id: "2"
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:13]:
```suggestion
```

**Review by isaac-pulumi — COMMENTED** [review]
changes address CMS bugs

File: content/events/hacktoberfest-2020/index-md.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+This year, Pulumi is participating in Hacktoberfest with three areas where the community at large can collaborate to build new open source capabilities:
+
+1. **Native Pulumi Providers** - Want Pulumi to work better with your favorite cloud services? We'll show you how to get started building providers that let Pulumi users take full advantage of these services.
+2. **Operator and Automation API** - Learn how to use Pulumi's new Kubernetes Superpowers to build truly automated deployments for your workloads.
+3. **Reusable Infrastructure Libraries** - Create new abstractions and infrastructure building blocks using Pulumi.
+
+## How to get started
+
+1. **[Register](https://hacktoberfest.digitalocean.com)** on the Hacktoberfest site for access to get credit for your open source contributions.
+2. [**Join the Pulumi Community Slack** ](https://slack.pulumi.com)and head over to the #Hacktoberfest channel.
```
Comment (leezen) [inline content/events/hacktoberfest-2020/index-md.md:36]: This is failing the build b/c of the space b/t `* ]`

File: content/events/hacktoberfest-2020/index-md.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
+---
+meta_desc: Join Pulumi for Hacktoberfest and help us build open source integrations, automation and libraries.
```
Comment (leezen) [inline content/events/hacktoberfest-2020/index-md.md:2]: This is causing the build to fail b/c the linter expects this to be <160 characters

File: content/events/hacktoberfest-2020/index-md.md (hunk 9 above)
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:36]:
```suggestion
2. [**Join the Pulumi Community Slack**](https://slack.pulumi.com)and head over to the #Hacktoberfest channel.
```

Comment (isaac-pulumi) [reply content/events/hacktoberfest-2020/index-md.md:36]: fixed

File: content/events/hacktoberfest-2020/index-md.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+  learn more about cloud programming, infrastructure as code, and many other
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:3]:
```suggestion
```

File: content/events/hacktoberfest-2020/index-md.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+  topics.
```
Comment (isaac-pulumi) [inline content/events/hacktoberfest-2020/index-md.md:4]:
```suggestion
```

File: content/events/hacktoberfest-2020/index-md.md (hunk 10 above)
Comment (isaac-pulumi) [reply content/events/hacktoberfest-2020/index-md.md:2]: addressed, now 99 characters.

File: content/events/hacktoberfest-2020/index-md.md (hunk 2 above)
Comment (isaac-pulumi) [reply content/events/hacktoberfest-2020/index-md.md]: fixed

**Comment by leezen** [comment]
Oddly, there seems to be a bug in the CMS where the file getting created is called `index-md.md` instead of `index.md` -- I'll rename the file in this branch.

**Review by leezen — APPROVED** [review]

**Pull request merged**